sdk/mixedreality/mixed-reality-authentication/src/mixedRealityStsClient.ts

```diff
@@ -1,19 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { AccessToken, AzureKeyCredential } from "@azure/core-auth";
-import { GetTokenOptions, MixedRealityStsClientOptions } from "./models/options";
-import {
-  InternalPipelineOptions,
-  TokenCredential,
-  bearerTokenAuthenticationPolicy,
-  createPipelineFromOptions,
-} from "@azure/core-http";
+import { InternalClientPipelineOptions } from "@azure/core-client";
+import { bearerTokenAuthenticationPolicy } from "@azure/core-rest-pipeline";
+import { TokenCredential, AccessToken, AzureKeyCredential } from "@azure/core-auth";
 import {
   MixedRealityStsRestClient,
-  MixedRealityStsRestClientGetTokenOptionalParams,
   MixedRealityStsRestClientOptionalParams,
+  GetTokenOptionalParams,
 } from "./generated";
+import { GetTokenOptions, MixedRealityStsClientOptions } from "./models/options";
 import { MixedRealityAccountKeyCredential } from "./models/auth";
 import { SDK_VERSION } from "./constants";
```

Comment:
You need to update the SDK_VERSION in the constants.ts file. It is still showing up as beta.1

Reply:
Thanks for spotting this